src/provisioner/provisioner.py

```diff
@@ -178,6 +201,8 @@ def cluster_bootstrap(cortx_conf_url: str = None):
 
         Log.info(f'Starting cluster bootstrap on {node_id}:{node_name}')
 
+        CortxProvisioner._update_deployment_status(
+            cortx_config_store, node_id, 'deployment')
```

Comment:
Use global defines for status and stage values

Reply:
Done